telemeta/util/backend.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+from haystack.backends.elasticsearch_backend import *
+import string
+import re
+
+regexSpacePunc = "[\\s"+re.escape(string.punctuation)+"]+"
+
+class CustomElasticBackend(ElasticsearchSearchBackend):
+
```

Comment:
please document the backend a little bit, because a custom backend can be a very crucial part of the software.